### DISCUSSION TITLE
Can i use remote Track that is received from peer-1 and set my localTrack from remote Track and send to peer-1 back?

### DISCUSSION THREAD

**Discussion by ManivishalBurra** [pion/webrtc#2560] [Q&A] [answered]
I am trying to understand PION implementation where remote peer can be achieved by peerconnection.onTrack() .so is there any way we can convert the remote track to my local track and addTrack peerconnection.addTrack(remoteTrack)?

**Comment by Sean-Der** [comment, marked as answer]
Hi @ManivishalBurra 

Nothing exists that can do that automatically (maybe we should provide that as a helper?)

[examples/broadcast](https://github.com/pion/webrtc/blob/master/examples/broadcast/main.go) does exactly what you are describing though! It reads from a `TrackRemote` and then writes to a `TrackLocal`

Another project I worked on recently does this [Broadcast Box](https://github.com/glimesh/broadcast-box). If you want to work on a re-usable helper that would be very useful!

**Reply by ManivishalBurra** [reply]
Thanks for the quick reply @Sean-Der . I'll go through broadcast and let you know.
